cypress/support/pages/room_board/pageRoomBoards.js

```diff
@@ -7,9 +7,54 @@ class RoomBoards {
 	static #mainPageArea = '[id="main-content"]';
 	static #roomBoardTitleOnPage = '[data-testid="board-title"]';
 	static #boardMenuIcon = '[data-testid="board-menu-icon"]';
+	static #btnBoardMenuActionRename = '[data-testid="kebab-menu-action-rename"]';
 	static #dialogAddMultiColumnBoard = '[data-testid="dialog-add-multi-column-board"]';
 	static #dialogAddSingleColumnBoard = '[data-testid="dialog-add-single-column-board"]';
 	static #boardLayoutDialogTitle = '[data-testid="board-layout-dialog-title"]';
+	static #breadCrumbToRoomNavigationFromBoard = '[data-testid="breadcrumb-1"]';
+	static #multicolumnBoardSelector = '[data-testid="board-tile-subtitle-0"]';
```

Comment:
make it similar in all instances in the code either its `multiColumn` or `multicolumn`